lib/ModelMixins/GltfMixin.ts

```diff
@@ -127,23 +170,33 @@ function GltfMixin<T extends Constructor<GltfModel>>(Base: T) {
       return super.shortReport;
     }
 
+    @computed
+    get modelEntity(): Entity {
+      const entity = this._modelEntity;
+      entity.position = new ConstantPositionProperty(this.cesiumPosition);
+      entity.orientation = new ConstantProperty(this.cesiumRotation);
+      entity.model = this.modelGraphics;
+      return entity;
```

Comment:
Maybe need to trigger `notifyRepaintRequired` here?

Reply:
I think the `notifyRepaintRequired()` will automatically happen when `mobx` recomputes the `mapItems` triggering a chain of events that [reaches here](https://github.com/TerriaJS/terriajs/blob/6c44081460f857a44f2daf84a42e5bae0871a924/lib/Models/Cesium.ts#L743).

Also i think the general rule is that we shouldn't access any other Terria resources from within the Catalog items because they are like data providers and must be independent of the rest of the infra(?)

Reply:
Thoughts @staffordsmith83?

Reply:
"Also i think the general rule is that we shouldn't access any other Terria resources from within the Catalog items because they are like data providers and must be independent of the rest of the infra(?)" oh yeah thats critical I think. Looked through that chain of events right you are!